analytics/views.py

```diff
@@ -1028,6 +1030,28 @@ def get_activity(request: HttpRequest) -> HttpResponse:
         context=dict(data=data, title=title, is_home=True),
     )
 
+def get_confirmations(type: int, object_ids: List[int]) -> List[Dict[str, Any]]:
+    days_to_activate = _properties[type].validity_in_days
+    lowest_datetime = timezone_now() - timedelta(days=days_to_activate)
+    confirmations = Confirmation.objects.filter(type=type, object_id__in=object_ids,
```

Comment:
Now that we're getting everything in the last 30 days, I think a more natural search might be to search by `Confirmation.content_type` rather than by `Confirmation.type` (and then get_confirmations would take a `content_type` as its argument rather than a `type`)

Reply:
hmm. We still need `type` for showing the number of days left for expiry right.

Reply:
yeah. I guess I could imagine doing the filter by content_type, and then determining the expiry by looking at the types of the returned Confirmation objects. But not a big deal either way.